index.js

```diff
@@ -158,8 +158,9 @@ NYC.prototype._prepGlobPatterns = function (patterns) {
 
 NYC.prototype.addFile = function (filename) {
   var relFile = path.relative(this.cwd, filename)
-  var source = stripBom(fs.readFileSync(filename, 'utf8'))
+  var source = this._readTranspiledSource(path.resolve(this.cwd, filename))
   var instrumentedSource = this._maybeInstrumentSource(source, filename, relFile)
```

Comment:
Better yet, use the profiling docs:

<img width="690" alt="screenshot 2016-01-18 19 44 34" src="https://cloud.githubusercontent.com/assets/4082216/12406542/f19242b4-be1b-11e5-945b-c11d7c55782d.png">

Fears alleviated! :ship: